src/application-manager.d.ts

```diff
@@ -35,7 +35,7 @@ declare interface Options {
 }
 
 // TODO: This needs to be moved to the correct module's typings
-declare interface Application {
+export declare interface Application {
```

Comment:
Is this definitely needed?

Reply:
ah actually that was when I stubbed the ApplicationManager and wanted to use the correct types. I later stubbed the ServiceManager, NetworkManager, VolumeManager, etc so no longer needed this export because I was not stubbing ApplicationManager.

I will remove this change!